_includes/content/ssh-add-keys-to-agent.md

```diff
@@ -0,0 +1,92 @@
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#mac-add-keys" data-group="mac">Mac</a></li>
+  <li><a data-toggle="tab" data-target="#win-add-keys" data-group="win">Windows</a></li>
+  <li><a data-toggle="tab" data-target="#linux-add-keys" data-group="linux">Linux</a></li>
+</ul>
+<div class="tab-content">
+<div id="mac-add-keys" class="tab-pane fade in active">
```

Comment:
I think you may need to add `markdown=1` to this div and any div whose contents contain Markdown. Then you don't need to do the  capture and then display.